packages/cli-kit/src/constants.ts

```diff
@@ -53,7 +53,8 @@ const constants = {
    */
   versions: {
     cliKit: async () => {
-      const packageJsonPath = (await findUp('package.json', {type: 'file'})) as string
+      const relativePath = pathJoin('node_modules', '@shopify', 'cli-kit', 'package.json')
```

Comment:
Correct me if I'm wrong, but `join` by default makes the path relative to the working directory. I'd change that to be relative to the directory that contains the module, `.../dist/constant.js`:
```js
pathJoin(__dirname, 'node_modules', '@shopify', 'cli-kit', 'package.json')
```

You can use [this function](https://github.com/Shopify/cli/blob/main/packages/cli-kit/src/path.ts#L37) to obtain the directory of the module: `moduleDirectory(import.meta.url)`